README.md

```diff
@@ -711,7 +711,7 @@ CoolConfig.new.port == "443" #=> true
 
 **IMPORTANT**: Values provided explicitly (via attribute writers) are not coerced. Coercion is only happening during the load phase.
 
-The following types are supported out-of-the-box: `:string`, `:integer`, `:float`, `:date`, `:datetime`, `:uri`, `:boolean`.
+The following types are supported out-of-the-box: `:string`, `:integer`, `:integer!`, `:float`, `:date`, `:datetime`, `:uri`, `:boolean`.
```

Comment:
Can we add some note on what's the difference between `:integer` and `:integer!`?